snap/validate.go

```diff
@@ -197,6 +299,21 @@ func ValidateApp(app *AppInfo) error {
 			return err
 		}
 	}
+
+	// Socket activation requires the "network-bind" plug
+	if len(app.Sockets) > 0 {
+		if _, ok := app.Plugs["network-bind"]; !ok {
+			return fmt.Errorf(`app with sockets must declare the "network-bind" plug`)
```

Comment:
Let's please word it as:

_"network-bind" interface plug is required when sockets are used_

Reply:
done